client/my-sites/patterns/home/index.tsx

```diff
@@ -30,94 +33,99 @@ export const PatternsHomePage = ( {
 	patternGallery: PatternGallery,
 }: PatternsHomePageProps ) => {
 	const locale = useLocale();
+	const isLoggedIn = useSelector( isUserLoggedIn );
 
 	const { data: categories } = usePatternCategories( locale, RENDERER_SITE_ID );
 	const { data: patterns } = usePatterns( locale, category );
 
 	return (
-		<Main isLoggedOut fullWidthLayout>
-			<DocumentHead title="WordPress Patterns" />
+		<>
+			{ isLoggedIn && <UniversalNavbarHeader isLoggedIn /> }
```

Comment:
The code just below is plenty of colors because I've changed the indentation but really the only thing that I've changed here is this new line.